.travis.yml

```diff
@@ -13,10 +13,12 @@ cache:
   yarn: true
 
 install:
+  - gem install rubocop
```

Comment:
@justin808 hmm, may be but don't like dependencies. Also, it seems rubocop is more kinda of standalone program now, like yarn and npm